File: beecrowd/1180.py
testes = int(input())
contador = posicao = 0
valores = input().split()
while True:
    if contador == 0:
        menor = int(valores[0])
    else:
        if int(valores[contador]) < menor:
            menor = int(valores[contador])
            posicao = contador
    contador += 1
    if contador == testes:
        break
print(f'Menor valor: {menor}')
print(f'Posicao: {posicao}')

File: beecrowd/1180.c
#include <stdio.h>

int main() {
    int n, i, menor, posicao;

    scanf("%d", &n);

    int vetor[n];

    for(i = 0; i < n; i++) {
        scanf("%d", &vetor[i]);
    }
    menor = vetor[0];
    posicao = 0;
    for(i = 1; i < n; i++) {
        if(vetor[i] < menor) {
            menor = vetor[i];
            posicao = i;
        }
    }
    printf("Menor valor: %d\n", menor);
    printf("Posicao: %d\n", posicao);
    return 0;
}